conf/cmi/core.entity_form_display.paragraph.hero.default.yml

```diff
@@ -74,13 +74,7 @@ content:
       size: 60
       placeholder: ''
     third_party_settings: {  }
-  field_display_search_field:
```

Comment:
This configuration rewrite needs to be added to `public/modules/custom/kaupunkitieto_config/config/rewrite`.